applications/jonny5/doc/balance_crawler.md

```diff
@@ -0,0 +1,20 @@
+/*
+Section: Jonny5
+Title: Balance crawler
+Language: en-US
+*/
+
+# Balance crawler
+Disconnects active per_minute calls when account balance drops below zero.
+
+## Crawler configuration options
+Document - `system_config/jonny5`
+Options:
+- `balance_crawler_enabled` - boolean, default `true`. Enable crawler.
+- `balance_crawler_cycle` - integer, milliseconds, default 60000 (1 minute). How often crawler check accounts balance.
```

Comment:
Can we change this to `balance_crawler_cycle_ms`. I prefer putting the time unit on the name so you can immediately tell what 60000 means.